web/src/notifications.js

```diff
@@ -61,19 +39,6 @@ export function get_notifications() {
     return notice_memory;
 }
 
-export function initialize({on_click_scroll_to_selected}) {
-    $(window).on("focus", () => {
-        for (const notice_mem_entry of notice_memory.values()) {
-            notice_mem_entry.obj.close();
-        }
-        notice_memory.clear();
-    });
-
-    update_notification_sound_source($("#user-notification-sound-audio"), user_settings);
-
-    register_click_handlers({on_click_scroll_to_selected});
-}
-
 export function update_notification_sound_source(container_elem, settings_object) {
```

Comment:
This function could possibly just be moved to a `web/src/audible_notifications.js` if we're going to rename this module to` `desktop_notifications.js` after extracting these two other modules; it's sorta the odd function left out.

Reply:
Sure. (It’s left out to dodge a `message_notifications` → `narrow` → `hashchange` → `admin` → `settings_org` → `message_notifications` cycle.)